login/db/user_db/CreateUserResponce.js

```diff
@@ -0,0 +1,12 @@
+const userResponse = require('../db_objects/user_db_objects/UserResponse.js');
+
+const userResponce = new userResponse.UserResponse();
+
+function createUserResponse(User) {
```

Comment:
Look into mapping between enteties.(you can also google automapper for node.js) This is implementend to have all of code like this(that maps from db objects to domain objects) easier to read and maintain.